lib/ansible/modules/identity/ipa/ipa_user.py

```diff
@@ -69,6 +69,10 @@
     description:
     - Posix Group ID
     version_added: 2.5
+  homedir:
+    description:
+    - Default home directory of the user
```

Comment:
Does the home directory need to already exist?
Do we need a `create_home` directory like in https://docs.ansible.com/ansible/latest/modules/user_module.html

Reply:
Looking at that link, it appears as though the functionality that I was trying to add is included/already in.  If someone can verify this, I'd be OK with closing this ticket.

Reply:
Actually, that's for a local user, not an IPA user.  So I'll look into what it would take to add a similar create_home flag/option.

Reply:
I looked at this a little bit, and I think it should work similar to how some of the other flags work (such as loginshell): if not specified, nothing should happen.  Let me know if this is not acceptable.